model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -203,13 +204,20 @@ message BundleApplication {
 }
 
 // An Application should be scheduled for execution after a delay.
+// Either an absolute timestamp or a relative timestamp can represent a
```

Comment:
I created a JIRA issue as follow up: [BEAM-8536](https://issues.apache.org/jira/browse/BEAM-8536). Maybe it would be better to separate this PR from java changes?